File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 2/ex10.c
#include <stdio.h>

main()
{
    printf("Custo ao consumidor\n\n");

    //vars
    float custoConsumidor, pDistribuidor, impostos, custoFabrica;

    //Entrada
    printf("Insira o custo de fabrica do veiculo: ");
    scanf("%f",&custoFabrica);

    //Processamento
    if ( custoFabrica <= 12000 )
    {
        pDistribuidor = custoFabrica * 0.05;
        impostos = 0;
    }
    else if ( custoFabrica <= 25000 )
    {
        pDistribuidor = custoFabrica * 0.1;
        impostos = custoFabrica * 0.15;
    }
    else
    {
        pDistribuidor = custoFabrica * 0.15;
        impostos = custoFabrica * 0.2;
    }

    custoConsumidor = custoFabrica + pDistribuidor + impostos;

    //Saída
    printf("Custo ao consumidor R$%.2f", custoConsumidor);

}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 2/src/Ex10.java
//Importando Scanenr
import java.util.Scanner;

public class Ex10{
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner input = new Scanner(System.in);
	
	System.out.println("Custo ao Consumidor\n\n");
	
	//vars
	double cFabrica, imposto, pDistribuidor, cConsumidor;
	
	//Entradas
	System.out.print("Insira o custo de fábrica do automóvel: ");
	cFabrica = input.nextDouble();
	
	//Processamento
	if ( cFabrica <= 12000 )
	{
		pDistribuidor = cFabrica * 0.05;
		imposto = 0;
	}
	else if ( cFabrica <= 25000 )
	{
		pDistribuidor = cFabrica * 0.1;
		imposto = cFabrica * 0.15;
	}
	else 
	{
		pDistribuidor = cFabrica * 0.15;
		imposto = cFabrica * 0.20;
	}
	cConsumidor = cFabrica + pDistribuidor + imposto;
	
	//Saída
	System.out.printf("Custo ao consumidor: R$ %.2f", cConsumidor);
	
}
	}